src/renderers/shared/reconciler/__tests__/ReactCompositeComponent-test.js

```diff
@@ -146,7 +146,7 @@ describe('ReactCompositeComponent', function() {
     var anchor = instance.getAnchor();
     var actualDOMAnchorNode = ReactDOM.findDOMNode(anchor);
```

Comment:
you shouldn't need `ReactDOM.findDOMNode` here, if I'm reading this right? IIRC since the ref is a non-composite, it should directly return the node